wordpress-mobile/WordPress-iOS
# Notifications row inline action groundwork

Part of #22473

This PR introduces the following updates:
- Add `InlineAction` type into `NotificationsTableViewCellContent.Style`.
- Dynamic notification row alignment:
  - Views align at the top when the text height surpasses the avatar's height.
  - In other cases, views are aligned at the center.

| Before | After |
| ------ | ------ |
| ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/9a2c9cea-01f4-4a05-99ff-7e4873399051) | ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/a58b3232-9a4b-48f6-b7db-5d1ca7208d99) |

## Test Instructions
1. Navigate to Notifications
2. **Verify** the row views are top aligned when the text height is higher than avatar's height
3. Rotate the device.
4. **Verify** the row views are center aligned when the text height is lower than avatar's height

## Regression Notes
1. Potential unintended areas of impact
N/A

5. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A


6. What automated tests I added (or what prevented me from doing so)
N/A

## PR submission checklist:
